itests/org.openhab.binding.mqtt.ruuvigateway.tests/itest.bndrun

```diff
@@ -0,0 +1,111 @@
+
+# We would like to use the "volatile" storage only
+-runblacklist: \
+	bnd.identity;id='org.openhab.core.storage.json'
+
+-runvm.mqtt: \
+	-Dio.netty.noUnsafe=true,\
+	-Dmqttbroker.port=${mqttbroker.port}
+
+#
+# done
+#
+-runbundles: \
+	org.apache.felix.http.servlet-api;version='[1.1.2,1.1.3)',\
+	org.eclipse.equinox.event;version='[1.4.300,1.4.301)',\
+	org.osgi.service.event;version='[1.4.0,1.4.1)',\
+	org.glassfish.hk2.external.javax.inject;version='[2.4.0,2.4.1)',\
+	org.hamcrest;version='[2.2.0,2.2.1)',\
+	org.jctools.core;version='[2.1.2,2.1.3)',\
+	org.opentest4j;version='[1.2.0,1.2.1)',\
+	jakarta.xml.bind-api;version='[2.3.3,2.3.4)',\
+	com.sun.xml.bind.jaxb-osgi;version='[2.3.3,2.3.4)',\
+	org.apache.servicemix.specs.activation-api-1.2.1;version='[1.2.1,1.2.2)',\
+	com.google.dagger;version='[2.27.0,2.27.1)',\
+	com.google.gson;version='[2.9.1,2.9.2)',\
+	com.hivemq.client.mqtt;version='[1.2.2,1.2.3)',\
+	io.netty.buffer;version='[4.1.72,4.1.73)',\
+	io.netty.codec;version='[4.1.72,4.1.73)',\
+	io.netty.codec-http;version='[4.1.59,4.1.60)',\
+	io.netty.codec-socks;version='[4.1.72,4.1.73)',\
+	io.netty.common;version='[4.1.72,4.1.73)',\
+	io.netty.handler;version='[4.1.72,4.1.73)',\
+	io.netty.handler-proxy;version='[4.1.72,4.1.73)',\
+	io.netty.resolver;version='[4.1.72,4.1.73)',\
+	io.netty.tcnative-classes;version='[2.0.46,2.0.47)',\
+	io.netty.transport;version='[4.1.72,4.1.73)',\
+	io.netty.transport-native-epoll;version='[4.1.59,4.1.60)',\
+	io.netty.transport-native-unix-common;version='[4.1.59,4.1.60)',\
+	io.reactivex.rxjava2.rxjava;version='[2.2.19,2.2.20)',\
+	jakarta.annotation-api;version='[2.0.0,2.0.1)',\
+	jakarta.inject.jakarta.inject-api;version='[2.0.0,2.0.1)',\
+	javax.measure.unit-api;version='[2.1.2,2.1.3)',\
+	junit-jupiter-api;version='[5.8.1,5.8.2)',\
+	junit-jupiter-engine;version='[5.8.1,5.8.2)',\
+	junit-platform-commons;version='[1.8.1,1.8.2)',\
+	junit-platform-engine;version='[1.8.1,1.8.2)',\
+	junit-platform-launcher;version='[1.8.1,1.8.2)',\
+	org.apache.aries.javax.jax.rs-api;version='[1.0.1,1.0.2)',\
+	org.apache.felix.scr;version='[2.1.30,2.1.31)',\
+	org.glassfish.hk2.osgi-resource-locator;version='[1.0.3,1.0.4)',\
+	org.jsr-305;version='[3.0.2,3.0.3)',\
+	org.osgi.util.function;version='[1.2.0,1.2.1)',\
+	org.osgi.util.promise;version='[1.2.0,1.2.1)',\
+	org.reactivestreams.reactive-streams;version='[1.0.3,1.0.4)',\
+	si-units;version='[2.1.0,2.1.1)',\
+	si.uom.si-quantity;version='[2.1.0,2.1.1)',\
+	tech.units.indriya;version='[2.1.2,2.1.3)',\
+	uom-lib-common;version='[2.1.0,2.1.1)',\
+	xstream;version='[1.4.19,1.4.20)',\
+	com.h2database.mvstore;version='[1.4.199,1.4.200)',\
+	com.zaxxer.HikariCP;version='[2.4.7,2.4.8)',\
+	io.dropwizard.metrics.core;version='[3.2.2,3.2.3)',\
+	io.netty.codec-mqtt;version='[4.1.72,4.1.73)',\
+	org.apache.commons.codec;version='[1.10.0,1.10.1)',\
+	org.apache.felix.configadmin;version='[1.9.24,1.9.25)',\
+	org.eclipse.jetty.http;version='[9.4.46,9.4.47)',\
```

Comment:
I can't fully verify this file, but I believe we are at Jetty 9.4.50 now, so please check versions here.

Reply:
Any tips to resolve these? Struggling with OSGi:

```
[ERROR] Resolution failed. Capabilities satisfying the following requirements could not be found:                                                          │[INFO] --- sortpom-maven-plugin:2.10.0:verify (sortpom-verify) @ org.openhab.core.bom.openhab-core-index ---
    [<<INITIAL>>]                                                                                                                                          │[INFO] Verifying file /home/salski/src/openhab-main/git/openhab-core/bom/openhab-core-index/pom.xml
      ⇒ osgi.identity: (osgi.identity=org.openhab.core.addon.xml)                                                                                          │[INFO]
    [osgi.cmpn version=7.0.0.201802012110]                                                                                                                 │[INFO] --- maven-install-plugin:2.5.2:install (default-install) @ org.openhab.core.bom.openhab-core-index ---
      ⇒ osgi.unresolvable: (&(must.not.resolve=*)(!(must.not.resolve=*)))        
```

Reply:
All resolved in https://github.com/openhab/openhab-addons/pull/13315/commits/37c2197c59750c1f8a48b5922f2394b5960fc798